.gitpod.yml

```
@@ -0,0 +1,8 @@
+image:
+  file: .gitpod.Dockerfile
+tasks:
+  - init: npm install
+    command: gulp
```

Please add a script to `package.json` that triggers `gulp` on `npm start`. Then, just use npm start in here.

```suggestion
    command: npm start
```